Bottom: 6fcc195c9d48654a5cea8b0e95224888590f51a4
Top:    236908b36395d49d5f6debc637dc17e64269d2ce
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:13:36 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b2539ca..7cdb152 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2686,7 +2686,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
